manifests/c/CPUID/CPU-Z/ARM64/1.01/CPUID.CPU-Z.ARM64.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: CPUID.CPU-Z.ARM64
+PackageVersion: "1.01"
+PackageLocale: en-US
+Publisher: CPUID, Inc.
+PublisherUrl: https://www.cpuid.com
+PublisherSupportUrl: https://www.cpuid.com/contact.html
+PrivacyUrl: https://www.cpuid.com/privacy-policy.html
+Author: Franck Delattre, CPUID
+PackageName: CPUID CPU-Z ARM64
+PackageUrl: https://www.cpuid.com/softwares/cpu-z.html
+License: Freeware
+LicenseUrl: https://www.cpuid.com/terms-of-service.html
+Copyright: CPUID ©2001-2022 - All website content subjected to copyright
+ShortDescription: System information software.
+Moniker: cpu-z

```

Comment:
This will cause a conflict with `CPUID.CPU-Z`. This moniker should be removed or changed